tools/docker/builder/ubuntu/bionic/Dockerfile

```diff
@@ -26,5 +26,34 @@ RUN apt-get update && apt-get install -y \
     python3-yaml \
     uuid-runtime \
     valgrind \
-    vim \
-    && rm -rf /var/lib/apt/lists/*
+    vim
+# AMBRIORIX dependencies
+RUN \
+    apt-get install -y \
+    bison \
+    flex \
+    libevent-dev \
+    libyajl-dev \
+    repo && \
+    echo "deb http://cz.archive.ubuntu.com/ubuntu eoan main universe" | tee -a  /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get install -y \
+    liburiparser-dev && \
+    rm -rf /var/lib/apt/lists/*
+WORKDIR ambiorix
+# Fetch and intall Bus Agnostic API libs, applications
+# "sed" is required to change from ssh to https protocols
+# https allows anonymous download, when ssh doesn't
+RUN \
+    repo init -u https://gitlab.com/soft.at.home/ambiorix/ambiorix.git && \
+    sed -i "s|ssh://git@gitlab.com/soft.at.home/ambiorix/|https://gitlab.com/soft.at.home/ambiorix/|g" .repo/manifests/default.xml && \
+    repo sync && \
```

Comment:
(nitpick) The Ambiorix project probably won't be updating regularly but I think we should still checkout a fixed revision. (release tag would be better but they didn't tag any commits). I am not sure how to do that in repo though.

Reply:
We should, but unfortunately AFAIK repo doesn't have support for that. I.e. we can check out a specific version of the manifest file, but since the manifest file just has branch names, we won't get a specific revision of all the subrepositories.

To solve this, I've been thinking about writing a script that parses a simple config file and checks stuff out accordingly. This could also be used for setting up the feeds in prplwrt and the layers in yocto, where we have exactly the same problem.